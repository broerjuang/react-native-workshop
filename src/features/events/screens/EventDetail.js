// @flow

import React, {Component} from 'react';
import {View, Text} from 'react-native';

class EventDetail extends Component<{}> {
  render() {
    return (
      <View>
        <Text>EventDetail</Text>
      </View>
    );
  }
}

export default EventDetail;
