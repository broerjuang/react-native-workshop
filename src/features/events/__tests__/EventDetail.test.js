import renderer from 'react-test-renderer';
import React from 'react';
import EventDetail from '../screens/EventDetail';
describe('container test', () => {
  it('should render LoginScreen corectly', () => {
    let component = renderer.create(<EventDetail />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
