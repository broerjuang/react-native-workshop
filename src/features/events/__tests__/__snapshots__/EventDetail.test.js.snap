// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`container test should render LoginScreen corectly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    EventDetail
  </Text>
</View>
`;
